deker/client.py

```diff
@@ -330,15 +330,20 @@ def calculate_storage_size(self, collection_name: str = "") -> StorageSize:
 
     def close(self) -> None:
         """Close client."""
-        if self.__adapter:
-            self.__adapter.close()
-            self.__adapter = None
-        if self.__factory:
-            self.__factory.close()
-            self.__factory = None
-        self.__is_closed = True
-        self.__ctx.is_closed = True
-        self.logger.info("Client is closed")
+        try:
+            if self.__adapter:
+                self.__adapter.close()
+                self.__adapter = None
+            if self.__factory:
+                self.__factory.close()
+                self.__factory = None
+            if self.__ctx:
+                self.__ctx.is_closed = True
+            self.__is_closed = True
+        except Exception as e:
```

Comment:
Same as in previous comment.

Reply:
[52b8348](https://github.com/openweathermap/deker/pull/15/commits/52b8348e12680f1d16548cbd43d8635fe2f109fb)